packages/data-types/src/types/v1/number.ts

```diff
@@ -0,0 +1,17 @@
+import { xLuceneFieldType } from '@terascope/types';
+import BaseType from '../base-type';
+import { ElasticSearchTypes } from '../../interfaces';
+
+export default class NumberClass extends BaseType {
+    toESMapping(_version?: number) {
+        return { mapping: { [this.field]: { type: 'double' as ElasticSearchTypes } } };
+    }
+
+    toGraphQL() {
+        return this._formatGql('Int');
```

Comment:
This needs to be float in order to avoid have a 32 bit integer size and to support floats.

```suggestion
        return this._formatGql('Float');
```